installation_tips/README.md

```diff
@@ -1,36 +1,53 @@
 ## Installation tips
 
-If you are not (yet) an expert in Python installations (conda vs pip, mananging environements, etc.),
-here we propose a simple recipe to install `spikeinterface` and several sorters inside an anaconda
-environment for windows/mac users.
+If you are not (yet) an expert in Python installations, a main difficulty is choosing the installation procedure.
+The main ideas you need to know before starting:
+ * python itself can be distributed and installed many many ways.
+ * python itself does not contain so many features for scientific computing you need to install "packages".
+   numpy, scipy, matplotlib, spikeinterface, ... are python packages that have a complicated dependency graph between then. "uv"
+ * packages  can be distributed and installed in several ways (pip, conda, uv, mamba, ...)
+ * installing many packages at once is challenging (because of their dependency graphs) so you need to do it in an "isolated environement"
+   to not destroy any previous installation. You need to see an "environment" as a sub installation in a dedicated folder.
+
+Choosing the installer + an environment manager + a package installer is a nightmare for beginners.
+The main options are:
+  * use "anaconda", which does everything. The most popular but bad idea because : ultra slow and aggressive licensing (not always free anymore)
+  * use python from the system or python.org + venv + pip : good idea for linux users.
+  * use "uv" : a new, fast and simple package manager. We recommend this for beginners on every os.
+
+Here we propose a step by step recipe for beginers based on "uv".
+We used to recommend installing with anaconda. It will be kept here for a while but we do not recommend it anymore.
+
 
 This environment will install:
  * spikeinterface full option
  * spikeinterface-gui
- * phy
- * tridesclous
+ * kilosort4
 
 Kilosort, Ironclust and HDSort are MATLAB based and need to be installed from source.
 
 ### Quick installation
 
-Steps:
+1. On macOS and Linux. Open a terminal and do
```

Comment:
Two comments:
1) I think it would be better to refer them to the uv installation guide. If it changes, the our docs will be outdated for no reason and they have os dependent paths:

https://docs.astral.sh/uv/getting-started/installation/

2) Why are we creating new requirements? for the stabel if uv installs from pip it should suffice, for the rolling, uv can install directly from github and that would cover that. 

In both cases it would use the pyproject.toml.

Reply:
I think this is a good point for these external tools!

Reply:
I think I do not want to overload the pyproject
The idea to have a working env for a beginner for a tutorial that contains spikeinterface but also external tools.
sigui sortingview ks.
The idea of having a entire and unqiue recipe and not refering to uv is to avoid a a benginer to understand evreything and compiling informations from several places.

Reply:
I don't understand, you don't need to add anything to the pyproject. The point of the proposal is to have less files on the repo.

Reply:
Here the requirements is bug : PySide6, jupyter.
I do not see this requirements as optional dependencies of spikeinterface but more a minimal stack for a beginner.
what would be the labels in pyproject.toml for theses 2 requirements files ?

Reply:
Fair enough, the right way would be to use the newly accepted [PEP 735](https://peps.python.org/pep-0735/) that covers just this but that is not available yet. 

I would suggest that it would be better for you (so you don't have to remember about dependencies in two places) if the requirements files only had the dependencies that are not in the pyprojec.toml there. Then you could do:

```bash
uv pip install spikeinterface
uv pip install `the_beginnger_required_stack.txt` 
```

But maybe you don't mind having these not working some time.